Turbolinks.xcodeproj/project.pbxproj

```diff
@@ -454,7 +454,6 @@
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 				ENABLE_NS_ASSERTIONS = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = arm64;
```

Comment:
https://developer.apple.com/documentation/technotes/tn3117-resolving-build-errors-for-apple-silicon

Reverting back to the default value:
`EXCLUDED_ARCHS` should be empty